PWGCF/EbyEFluctuations/Tasks/MeanPtFlucIdentified.cxx

```diff
@@ -352,63 +384,116 @@ struct meanPtFlucId {
+         (std::fabs(track.tpcNSigmaPi() + cfgMcTpcShiftPi) < cfgCutNSig2 && p > cfgCutPiP1 && p <= cfgCutPiP2))) {
+      if (abs(track.rapidity(massPi)) < cfgCutRap) {
         return true;
+      }
     }
     return false;
   }
 
-  // PID selction cuts for Kaons
+  // PID selction cuts for Low momentum Kaons
   template <typename T>
-  bool selKaons(T const& track, double innerParam)
+  bool selLowKa(T const& track, double p)
   {
-    if (track.pt() >= cfgCutKaPtMin && (((!track.hasTOF()) && std::abs(track.tpcNSigmaEl()) > cfgCutNSigTpcEl &&
-                                         ((std::abs(track.tpcNSigmaKa()) < cfgCutNSig3 && innerParam <= cfgCutKaP1) || (std::abs(track.tpcNSigmaKa()) < cfgCutNSig2 && innerParam > cfgCutKaP1 && innerParam <= cfgCutKaP2))) ||
-                                        (track.hasTOF() && std::abs(track.tpcNSigmaKa()) < cfgCutNSig3 && std::abs(track.tofNSigmaEl()) > cfgCutNSigTofEl &&
-                                         ((std::abs(track.tofNSigmaKa()) < cfgCutNSig3 && track.p() <= cfgCutKaP3) || (std::abs(track.tofNSigmaKa()) < cfgCutNSig2 && track.p() > cfgCutKaP3))))) {
-      if (abs(track.rapidity(massKa)) < cfgCutRap)
+    if (track.pt() >= cfgCutKaPtMin && std::fabs(track.tpcNSigmaPi() + cfgMcTpcShiftPi) > 3 && std::fabs(track.tpcNSigmaPr() + cfgMcTpcShiftPr) > 3 && track.p() <= cfgCutKaThrsldP &&
+        ((std::fabs(track.tpcNSigmaKa() + cfgMcTpcShiftKa) < cfgCutNSig3 && p <= cfgCutKaP1) ||
+         (std::fabs(track.tpcNSigmaKa() + cfgMcTpcShiftKa) < cfgCutNSig2 && p > cfgCutKaP1 && p <= cfgCutKaP2))) {
+      if (abs(track.rapidity(massKa)) < cfgCutRap) {
         return true;
+      }
     }
 
     return false;
   }
 
-  // PID selction cuts for Protons
+  // PID selction cuts for Low momentum Protons
   template <typename T>
-  bool selProtons(T const& track, double innerParam)
+  bool selLowPr(T const& track, double p)
   {
-    if (track.pt() >= cfgCutPrPtMin && (((!track.hasTOF()) && std::abs(track.tpcNSigmaEl()) > cfgCutNSigTpcEl &&
-                                         ((std::abs(track.tpcNSigmaPr()) < cfgCutNSig3 && innerParam <= cfgCutPrP1) || (std::abs(track.tpcNSigmaPr()) < cfgCutNSig2 && innerParam > cfgCutPrP1 && innerParam <= cfgCutPrP2))) ||
-                                        (track.hasTOF() && std::abs(track.tpcNSigmaPr()) < cfgCutNSig3 && std::abs(track.tofNSigmaEl()) > cfgCutNSigTofEl && std::abs(track.tofNSigmaPr()) < cfgCutNSig3))) {
-      if (abs(track.rapidity(massPr)) < cfgCutRap)
+    if (track.pt() >= cfgCutPrPtMin && std::fabs(track.tpcNSigmaKa() + cfgMcTpcShiftKa) > 3 && std::fabs(track.tpcNSigmaPi() + cfgMcTpcShiftPi) > 3 && track.p() <= cfgCutPrThrsldP &&
+        ((std::fabs(track.tpcNSigmaPr() + cfgMcTpcShiftPr) < cfgCutNSig3 && p <= cfgCutPrP1) ||
+         (std::fabs(track.tpcNSigmaPr() + cfgMcTpcShiftPr) < cfgCutNSig2 && p > cfgCutPrP1 && p <= cfgCutPrP2))) {
+      if (abs(track.rapidity(massPr)) < cfgCutRap) {
```

Comment:
Replace `abs` with `std::abs` everywhere. `abs` is known to be rounding to `int`.

@victor-gonzalez Please try to clear these issues in PWGCF systematically.

Reply:
@TGahlaut1 the latest `abs` on these above three functions are still pending to move to `std::abs`